schema/mvp.json

```diff
@@ -0,0 +1,326 @@
+      }
+    },
+    {
+      "name": "reference_file",
+      "columns": [
+        { "name": "reference_file_id", "datatype": "string" },
+        { "name": "version", "datatype": "timestamp" },
+        { "name": "content", "datatype": "string" },
+        { "name": "file_id", "datatype": "fileref" },
+        { "name": "source_file_id", "datatype": "string" },
+        { "name": "source_file_version", "datatype": "timestamp" }
+      ],
+      "primaryKey": ["reference_file_id"],
+      "partitionMode": "date",
+      "datePartitionOptions": {
+        "column": "version"
+      }
+    },
+    {
+      "name": "sequence_file",
+      "columns": [
+        { "name": "sequence_file_id", "datatype": "string" },
+        { "name": "version", "datatype": "timestamp" },
+        { "name": "content", "datatype": "string" },
+        { "name": "file_id", "datatype": "fileref" },
+        { "name": "source_file_id", "datatype": "string" },
+        { "name": "source_file_version", "datatype": "timestamp" }
+      ],
+      "primaryKey": ["sequence_file_id"],
+      "partitionMode": "date",
+      "datePartitionOptions": {
+        "column": "version"
+      }
+    },
+    {
+      "name": "supplementary_file",
+      "columns": [
+        { "name": "supplementary_file_id", "datatype": "string" },
+        { "name": "version", "datatype": "timestamp" },
+        { "name": "content", "datatype": "string" },
+        { "name": "file_id", "datatype": "fileref" },
+        { "name": "source_file_id", "datatype": "string" },
+        { "name": "source_file_version", "datatype": "timestamp" }
+      ],
+      "primaryKey": ["supplementary_file_id"],
+      "partitionMode": "date",
+      "datePartitionOptions": {
+        "column": "version"
+      }
+    },
+    {
+      "name": "process",
+      "columns": [
+        { "name": "process_id", "datatype": "string" },
+        { "name": "version", "datatype": "timestamp" },
+        { "name": "content", "datatype": "string" }
+      ],
+      "primaryKey": ["process_id"],
+      "partitionMode": "date",
+      "datePartitionOptions": {
+        "column": "version"
+      }
+    },
+    {
+      "name": "analysis_process",
+      "columns": [
+        { "name": "analysis_process_id", "datatype": "string" },
+        { "name": "version", "datatype": "timestamp" },
+        { "name": "content", "datatype": "string" }
+      ],
+      "primaryKey": ["analysis_process_id"],
+      "partitionMode": "date",
+      "datePartitionOptions": {
+        "column": "version"
+      }
+    },
+    {
+      "name": "project",
+      "columns": [
+        { "name": "project_id", "datatype": "string" },
+        { "name": "version", "datatype": "timestamp" },
+        { "name": "content", "datatype": "string" }
+      ],
+      "primaryKey": ["project_id"],
+      "partitionMode": "date",
+      "datePartitionOptions": {
+        "column": "version"
+      }
+    },
+    {
+      "name": "protocol",
```

Comment:
Unclear if this table is needed: https://github.com/HumanCellAtlas/metadata-schema/issues/1269#issuecomment-623534894

Reply:
Answer: we're still not sure, but having the extra table doesn't hurt anything